src/current/v23.2/log-sql-statistics-to-datadog.md

```diff
@@ -82,12 +89,33 @@ SET CLUSTER SETTING sql.telemetry.query_sampling.max_event_frequency = 100000;
 The `sql.telemetry.query_sampling.max_event_frequency` cluster setting and the `buffering` options in the `logs.yaml` control how many events are emitted to Datadog and that can be potentially dropped. Adjust this setting and these options according to your workload, depending on the size of events and the queries per second (QPS) observed through monitoring.
 {{site.data.alerts.end}}
 
-## Step 4. Monitor TELEMETRY logs in Datadog
+## Step 4. Configure CockroachDB to emit query and transaction events (optional)
+
+Enable the [`sql.telemetry.query_sampling.enabled` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-query-sampling-enabled) so that executed queries and transactions will emit an event on the telemetry [logging channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels):
```

Comment:
love the anchor link!

in that case I guess the lack of it in the other place was intentional

hopefully that other setting gets made public ASAP though if we're publishing it in docs